.github/workflows/element-with-modules.yml

```
@@ -0,0 +1,351 @@
+name: Element with modules
+
+on:
+  push:
+    branches: # we want to build the widgets from the push into Element staging and publish a main container
+      - main
+    tags: # we want to build the stable widgets into stable Element and publish the container
+      - '@nordeck/element*module*'
+  pull_request: # we want to build the widgets from the PR into Element develop & staging
+  schedule: # we want to build the stable widgets into stable Element and publish the container, plus a verification of all dev versions on a daily basis
+    - cron: '0 5 * * 1-5'
+  workflow_dispatch: # we build whatever the user inputs and publish the container
+    inputs:
+      element:
+        description: 'Element Web version (git ref)'
+        required: true
+        type: 'string'
+      guestModule:
+        description: 'Element Guest module version (release tag/git ref/none)'
+        required: true
+        type: 'string'
+      opendeskModule:
+        description: 'Element openDesk module version (release tag/git ref/none)'
+        required: true
+        type: 'string'
+      lifecycleModule:
+        description: 'Element Widget Lifecycle module version (release tag/git ref/none)'
+        required: true
+        type: 'string'
+      togglesModule:
+        description: 'Element Widget Toggles module version (release tag/git ref/none)'
+        required: true
+        type: 'string'
+
+jobs:
+  determine-versions:
+    name: Determine the versions for which to build containers
+    runs-on: ubuntu-latest
+    timeout-minutes: 15
+    permissions:
+      contents: read
+    outputs:
+      matrix: ${{ env.MATRIX }}
+    steps:
+      - name: Determine Element Web version
```

```suggestion
      - name: Determine Element Web versions
```

This is finding out the versions depending on the event or override by the input, right? I was first a bit confused, because it said „version“. Applies to other jobs as well.

---

Hm, it calculates `$VERSION_ELEMENT` specifically but yeah I guess it evolved to versions.